R/dot-net-wrapper.R

```diff
@@ -8,7 +8,7 @@
 #'   \item{wrapProperty}{Simple way to wrap a get;set; .NET property}
 #'   \item{wrapReadOnlyProperty}{Simple way to wrap a get; .NET Read-Only property}
 #'   }
-#'
+#' @keywords internal
 DotNetWrapper <- R6::R6Class(
   "DotNetWrapper",
   inherit = Printable,

```

Comment:
@IndrajeetPatil @PavelBal @msevestre The class would enable ospsuite-R users to encapsulate further .NET functionalities from PK-Sim. Even though it's very unlikely that somebody will do that: should we nevertheless keep the functionality in the docu?

Reply:
I think that's fair. We stand nothing to lose by exporting it. I will undo this change.